README.md

```diff
@@ -1,14 +1,32 @@
-# ImagePicker
+# ImagePicker :camera:
 
-[![CI Status](http://img.shields.io/travis/hyperoslo/ImagePicker.svg?style=flat)](https://travis-ci.org/hyperoslo/ImagePicker)
 [![Version](https://img.shields.io/cocoapods/v/ImagePicker.svg?style=flat)](http://cocoadocs.org/docsets/ImagePicker)
 [![License](https://img.shields.io/cocoapods/l/ImagePicker.svg?style=flat)](http://cocoadocs.org/docsets/ImagePicker)
 [![Platform](https://img.shields.io/cocoapods/p/ImagePicker.svg?style=flat)](http://cocoadocs.org/docsets/ImagePicker)
 
+## Description
+
+![ImagePicker](https://github.com/hyperoslo/ImagePicker/master/Resources/ImagePickerPresentation.png)
+
+**ImagePicker** is the all in one camera solution for your iOS app. Let your users select images from the library or take pictures at the same time with this component. Get notified of every event and get all the UI work for free.
```

Comment:
**ImagePicker** is the all in one camera solution for your iOS app

Maybe?

**ImagePicker** is an all-in-one camera solution for your iOS app